content/References/IPv6.md

```diff
@@ -0,0 +1,239 @@
+---
+title: "Understanding IPv6"
+description: "Provides information general information on IPv6 and implementation guidance."
+weight: 20
+tags:
+- network
+- interfaces
+- ipv6
+---
+
+## What is IPv6?
+IPv6, referred to as the Internet protocol next generation (IPng), is a 128-bit alphanumeric address organized into eight groups of 16 bits with each group expressed as four-digit hexadecimal numbers separated by colons.
```

Comment:
IPng was the name for IPv6 before it became a standard.  No one has used that term for 30 years.

Reply:
removed this content